<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLCreateWioDlg</class>
 <widget class="QDialog" name="FLCreateWioDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new WIO</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>WIO</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Signaling</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>WIO  Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MAC ADDRESS</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="leName"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLineEdit" name="lineEdit">
        <property name="inputMask">
         <string>NN:NN:NN:NN:NN:NN:NN:NN</string>
        </property>
        <property name="text">
         <string>FF:ff:ff:ff:ff:ff:ff:ff</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Undefined</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Continues</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Temporal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>March Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>California</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="2" colspan="3">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Output</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Number:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>174</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FLCreateWioDlg</receiver>
   <slot>on_model_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_model_changed(int)</slot>
 </slots>
</ui>
